vhpidirect/quickstart/logicvector/run.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env sh
+
+cd "$(dirname $0)"
+
+set -e
+
+ghdl -a tb.vhd
+ghdl -e -Wl,-I./ -Wl,main.c tb &&
+./tb
+#valgrind --leak-check=full --show-leak-kinds=all --track-origins=yes --verbose --log-file=valgrind-out.txt ./tb &&
```

Comment:
This does not fit here. However, I think it might be interesting to have an example/article about how to check memory leaks with valgrind. Ref ghdl/ghdl#1226.

Reply:
Sure. It's a general learning point though so I dont know how thoroughly I would want to document it in ghdl's docs... also, my use case was very simple:

Perhaps the example of checking memory should be linked to the full ghdl.h, because that is why I first learnt it. A quick section on its output, meaning and maybe an [external link](https://stackoverflow.com/a/44989219) to where I first learnt about it.

Reply:
In fact, just a markdown file in a feature branch of yours with a few snippets and references is enough. I mostly meant it for you not to lose this tiny bits of knowledge. If your ever feel confident about those notes, you can upstream them.